ern-composite-gen/test/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Was this file added accidentally?

Reply:
@friederbluemle
while running the `yarn run test:unit` we see this file created as untracked changes.
```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	ern-composite-gen/test/package.json
```
I'll remove this file